pkg/operation/cloudbotanist/gcpbotanist/controlplane.go

```diff
@@ -40,10 +42,29 @@ func (b *GCPBotanist) GenerateCloudProviderConfig() (string, error) {
 		networkName = b.Shoot.SeedNamespace
 	}
 
+	var (
+		subnetID       = "subnet_internal"
+		subNetworkName = ""
+	)
+	tf, err := b.NewShootTerraformer(common.TerraformerPurposeInfra)
+	if err != nil {
+		return "", err
+	}
+	stateVariables, err := tf.GetStateOutputVariables(subnetID)
+	if err != nil {
+		if terraformer.IsVariablesNotFoundError(err) {
+			b.Logger.Infof("Skipping explicit GCP subnet creation for internal loadbalancers because subnet_internal variable has not been found in the Terraform state.")
+		} else {
+			return "", err
+		}
```

Comment:
Refactor to make it better readable:

```golang
	if err != nil {
		if !terraformer.IsVariablesNotFoundError(err) {
			return "", err
		}

		b.Logger.Debugf("Skipping explicit GCP subnet creation for internal loadbalancers because subnet_internal variable has not been found in the Terraform state.")
	}
```

Note the change from `Infof` to `Debugf`